packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -200,7 +200,7 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
     try {
       kibanaRequest = CoreKibanaRequest.from(request, routeSchemas);
     } catch (error) {
-      this.log.error(`400 Bad Request - ${request.path}`, {
```

Comment:
I'm not sure if the path can be considered PII. I'll remove it just in case, and we can add it back later on if we deem it safe.